test/sample_graph.csv

```diff
@@ -0,0 +1,3 @@
+key,oid,local_edges,foreign_edges
```

Comment:
Let's chat later about the format here. After giving it some thought, I think this format is probably too restrictive.